source/extensions/tracers/opentelemetry/resource_detectors/dynatrace/dynatrace_resource_detector.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "envoy/extensions/tracers/opentelemetry/resource_detectors/v3/dynatrace_resource_detector.pb.h"
+
+#include "source/common/common/logger.h"
+#include "source/extensions/tracers/opentelemetry/resource_detectors/dynatrace/dynatrace_metadata_file_reader.h"
+#include "source/extensions/tracers/opentelemetry/resource_detectors/resource_detector.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+const std::array<std::string, 3> DT_METADATA_FILES = {
```

Comment:
```suggestion
constexpr std::array<std::string, 3> DT_METADATA_FILES = {
```
to make it compile time const

Reply:
Alright!. I had to change to `absl::string_view` as it was complaining string was not a literal type.